src/main/resources/anniv5Resources/localization/eng/showmanpack/Cardstrings.json

```diff
@@ -0,0 +1,43 @@
+{
+  "${ModID}:Dazzle": {
+    "NAME": "Dazzle",
+    "DESCRIPTION": "Deal !D! damage. NL Apply !M! Weak."
+  },"${ModID}:Amaze": {
+  "NAME": "Amaze",
+  "DESCRIPTION": "ALL enemies lose !M! HP for each stack of Weak they have."
+},"${ModID}:Grandstand": {
+  "NAME": "Grandstand",
+  "DESCRIPTION": "Gain !B! Block. NL Next turn, gain [E] .",
+  "UPGRADE_DESCRIPTION":  "Gain !B! Block. NL Next turn, gain [E] [E] ."
+},"${ModID}:DramaticExit": {
+  "NAME": "Disappear",
+  "DESCRIPTION": "Retain. NL Gain !B! Block. NL Exhaust."
+},"${ModID}:ForMyNextTrick": {
+  "NAME": "For My Next Trick...",
+  "DESCRIPTION": "Add a random Power card into your hand. NL Whenever you play a Power, gain !M! Block.",
```

Comment:
```suggestion
  "DESCRIPTION": "Add a random Power into your hand. NL Whenever you play a Power, gain !M! Block.",
```

Reply:
Worded to be consistent with creative AI